core/src/main/java/nl/inl/blacklab/contentstore/ContentStoreDirUtf8.java

```diff
@@ -474,6 +477,10 @@ public void addToBlock(String contentPart) {
         currentBlockContents.append(contentPart);
     }
 
+    public void addToBlock(char[] contentPart, int start, int length) {
+        currentBlockContents.append(contentPart);
+    }
+
```

Comment:
Waarom negeren we hier de start en length. Misschien dat in blacklab zelf het zo loopt dat die parameters altijd [0, part.length - 1] zijn maar het zou niet moeten volgens mij.

Reply:
Als het goed is gefixt.